.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
I don't think there is much reason to include this file in the absence of any docker images.

Reply:
Removed.